### PR TITLE
frontend release december

### PR DESCRIPTION
**Pull request by enikonemeth**

### CODE REVIEW

**Review by herflis — APPROVED** [review]

File: _updates/2019-12-11-december.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+- New dialogs have been created for Contextmenu (Approve/Reject, Versions, Check In/Out, Delete)🆕 [#379](https://github.com/sensenet/sn-client/issues/379)
+- The bug of upload the same file right after cancellation has been solved 🃏 [#491](https://github.com/sensenet/sn-client/issues/491)
+- Versioninfo item has removed from the drawer ✂️ [#425](https://github.com/sensenet/sn-client/issues/425)
+- Schema is now automatically updated on Save 🔄 [#489](https://github.com/sensenet/sn-client/issues/489)
+
+## Packages
+
+### @sensenet/controls-react@3.3.2
+- Content name is 'undefined' in the notification after modification bug has been fixed 🎫 [#450](https://github.com/sensenet/sn-client/issues/450)
+- The deafult text color has been changed to black in dark mode cause it was not visible on white background ⚫️⚪️ [#490](https://github.com/sensenet/sn-client/issues/490)
```
Comment (zoltanbedi) [inline _updates/2019-12-11-december.md]:
```suggestion
- The deafult text color for rich text editor has been changed to black in dark mode cause it was not visible on white background ⚫️⚪️ [#490](https://github.com/sensenet/sn-client/issues/490)
```

File: _updates/2019-12-11-december.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  [sn-react-calendar](https://sn-react-calendar.netlify.com)  
+  [sn-react-browser](https://sn-react-browser.netlify.com)  
+  [sn-react-tasklist](https://sn-react-tasklist.netlify.com)  
+  [sn-react-usersearch](https://sn-react-usersearch.netlify.com)
+
+  Thank you for the contributors:  
+  [blaskodaniel](https://github.com/blaskodaniel),
+  [Khegyi](https://github.com/Khegyi),
+  [VargaJoe](https://github.com/VargaJoe)
+  and [B3zo0](https://github.com/B3zo0)
```
Comment (zoltanbedi) [inline _updates/2019-12-11-december.md]:
```suggestion
```
I didn't work on any example app.

File: _updates/2019-12-11-december.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+## Admin-ui@1.4.0
+
+- We had to provide a hotfix cause there was a critical bug in admin-ui that prevented the user from uploading. 😱
+- Custom actions for CRUD have been created and some bugs have been fixed regarding this ✔️
```
Comment (zoltanbedi) [inline _updates/2019-12-11-december.md]: I think this is already in the previous release. Isn't it?

**Review by zoltanbedi — CHANGES_REQUESTED** [review]

File: _updates/2019-12-11-december.md (hunk 3 above)
Comment (enikonemeth) [reply _updates/2019-12-11-december.md]: I think yes :) Removed!

**Review by zoltanbedi — APPROVED** [review]

**Pull request merged**